### PR TITLE
Mfw 5903 -  added bypass settings schema  configurations

### PR DESCRIPTION
**Pull request by vishal-malji**
Added Bypass scheme 

Bypasss Test case result
image
Bypass Test Json : 
```{
    "bypass": {
        "rules": [
            {
                "action": {
                    "type": "BYPASS"
                },
                "conditions": [
                    {
                        "op": "==",
                        "port_protocol": "17",
                        "type": "DESTINATION_PORT",
                        "value": "89"
                    }
                ],
                "description": "bypass test rule",
                "enabled": true,
                "ruleId": "9c9b22c1-3eb7-4218-ac46-de5d2eb4dc18",
                "log": "bypass test log"
            }
        ]
    }
}

### CODE REVIEW

**Review by Utkarsh51 — CHANGES_REQUESTED** [review]
Add Vscode task for this

**Review by Utkarsh51 — APPROVED** [review]

File: v1/bypass/bypass_test.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                    {
+                        "op": "==",
+                        "port_protocol": "17",
+                        "type": "DESTINATION_PORT",
+                        "value": "89"
+                    }
+                ],
+                "description": "bypass test rule",
+                "enabled": true,
+                "ruleId": "9c9b22c1-3eb7-4218-ac46-de5d2eb4dc18"
```
Comment (Utkarsh51) [inline v1/bypass/bypass_test.json:18]: log property is missing

File: v1/bypass/bypass_schema.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            "properties": {
+                "description": {
+                    "type": "string"
+                },
+                "enabled": {
+                    "type": "boolean"
+                },
+                "ruleId": {
+                    "type": "string"
+                },
```
Comment (Utkarsh51) [inline v1/bypass/bypass_schema.json:28]: log property is missing

File: v1/bypass/bypass_schema.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                        "SOURCE_INTERFACE_TYPE",
+                        "SOURCE_INTERFACE_ZONE",
+                        "SOURCE_PORT",
+                        "IP_PROTOCOL"
+                    ]
+                },
+                "value": {
+                    "type": "string"
+                },
+                "op": {
```
Comment (Utkarsh51) [inline v1/bypass/bypass_schema.json:70]: port_protocol is missing from condition schema

File: v1/bypass/bypass_test.json (hunk 1 above)
Comment (vishal-malji) [reply v1/bypass/bypass_test.json]: added

File: v1/bypass/bypass_schema.json (hunk 3 above)
Comment (vishal-malji) [reply v1/bypass/bypass_schema.json:73]: added

File: v1/bypass/bypass_schema.json (hunk 2 above)
Comment (vishal-malji) [reply v1/bypass/bypass_schema.json:28]: added

**Review by AlistairJohnParker — APPROVED** [review]

**Review by Utkarsh51 — APPROVED** [review]

**Pull request merged**